package.json

```
@@ -13,46 +13,48 @@
     },
     "homepage": "https://github.com/fluid-project/handbook.floeproject.org#readme",
     "scripts": {
-        "build": "npm-run-all -l clean build:*",
-        "build:webpack": "mix --production",
+        "build": "npm-run-all -l clean build:eleventy",
         "build:eleventy": "cross-env NODE_ENV=production eleventy",
-        "clean": "rimraf dist",
+        "clean": "rimraf _site",
         "dev": "cross-env NODE_ENV=development eleventy --serve",
-        "lint": "fluid-lint-all",
-        "start": "npm-run-all -l clean -p start:*",
-        "start:webpack": "mix watch",
-        "start:eleventy": "npm run dev",
+        "lint": "run-s -l lint:*",
+        "lint:css": "stylelint \"**/*.css\"",
```

This should be changed:

```suggestion
        "lint:css": "stylelint \"**/*.scss\"",
```

---

@chosww I noticed you changed the script name— I've purposely kept the script name as `lint:css` even when Sass is used just for consistency. I wonder if maybe we should move to using `lint:scripts` and `lint:styles` instead?